app/views/errors/internal_server_error.html.erb

```diff
@@ -0,0 +1,5 @@
+<main id="content" role="main">
```

Comment:
Should you set the page title here? Usually something like "Page not found - 404 - data.gov.uk"